File: go/pkg/types/account.go
package types

type Account struct {
	DropletLimit        int    `json:"droplet_limit"`
	FloatingIPLimit     int    `json:"floating_ip_limit"`
	VolumeLimit         int    `json:"volume_limit"`
	Email               string `json:"email"`
	UUID                string `json:"uuid"`
	EmailVerified       bool   `json:"email_verified"`
	Status              string `json:"status"`
	StatusMessage       string `json:"status_message"`
	Team                *Team  `json:"team,omitempty"`
}

type Team struct {
	UUID string `json:"uuid"`
	Name string `json:"name"`
}

type AccountResponse struct {
	Account Account `json:"account"`
}

File: rust/crates/do-types/src/account.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct Account {
    pub droplet_limit: u32,
    pub floating_ip_limit: u32,
    pub volume_limit: u32,
    pub email: String,
    pub uuid: String,
    pub email_verified: bool,
    pub status: String,
    pub status_message: String,
    #[serde(skip_serializing_if = "Option::is_none")]
    pub team: Option<Team>,
}

#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct Team {
    pub uuid: String,
    pub name: String,
}

#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct AccountResponse {
    pub account: Account,
}